Update doc: iOS does not support destination middleware (#5078)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/ios/middleware.md

strat: ios
---

Middlewares are simple functions invoked by the Segment libraries, which give you a way to add information to the events you collect using the Segment SDKs. They can be used to monitor, modify, or reject events. Source Middlewares are available on `analytics-ios` 3.6.0 and later. 

You can access the middleware API in both Objective-C and Swift.

> info ""

> > **Note**: Destination middleware is **not** available for iOS.

Reviewer: ```suggestion
> **Note**: Destination Middlewares are **not** available for iOS.
```

> > **Note**: Destination middleware is **not** available for iOS.

Reviewer: ```suggestion
> **Note**: Destination Middlewares are **not** available for iOS.
```

    turnScreenIntoTrack,
    enforceEventTaxonomy,
    customizeAllTrackCalls,
    dropSpecificEvents,

>     blockScreenCallsToAmplitude

Reviewer: ```suggestion
    blockScreenCallsToAmplitude,
```

    turnScreenIntoTrack,
    enforceEventTaxonomy,
    customizeAllTrackCalls,
    dropSpecificEvents,

>     blockScreenCallsToAmplitude

Reviewer: ```suggestion
    blockScreenCallsToAmplitude,
```

#### Block specific call types to a specific destination

> The following example blocks only screen calls from reaching the Amplitude destination. 

Reviewer: ```suggestion
The following example blocks only screen calls from reaching the Amplitude destination.
```
